<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_artistName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Artist Name: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artistName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_Genre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genre:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play </string>
   </property>
  </widget>
  <widget class="QPushButton" name="PurchaseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose A Track!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Song">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Song:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genreName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="songName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="LikeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Song_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Purchased Songs:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="purchasedSongs">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Song_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search Result:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResults">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
